website/views/shared/profiles/achievements.jade

```diff
@@ -1,5 +1,8 @@
 mixin simpleAchiev(achiev)
-  div(ng-init='earnedClass = achiev.earned ? "" : "muted"')&attributes(attributes)
+  div(ng-init='earnedClass = achiev.earned ? "" : "muted"', 
+      data-popover-html='<div class="{{earnedClass}}">{{achiev.title}}<hr>{{achiev.text}}</div>',
```

Comment:
I had the popover on the same placement, but the clicked one is about 3 pixel lower than the "mouseenter", but I could change it back